data/brands/amenity/pharmacy.json

```
@@ -4852,6 +4852,57 @@
         "name:en": "Cowell",
         "name:zh": "高济"
       }
+    },
+    {
+      "displayName": "Nataly Pharm",
+      "locationSet": {"include": ["am"]},
+      "tags": {
+        "amenity": "pharmacy",
+        "brand": "Նատալի ֆարմ",
+        "brand:en": "Nataly Pharm",
+        "brand:hy": "Նատալի ֆարմ",
+        "brand:ru": "Натали фарм",
+        "healthcare": "pharmacy",
+        "name": "Նատալի ֆարմ",
+        "name:en": "Nataly Pharm",
+        "name:hy": "Նատալի ֆարմ",
+        "name:ru": "Натали фарм",
+        "website": "https://natalipharm.am"
```

```suggestion
        "name:ru": "Натали фарм"
```

Websites don't belong in NSI.

---

Few brands are missing wikidata ID. Will add them to wikidata and update the PR with ids and no website.